plugins/aws/resoto_plugin_aws/resource/athena.py

```diff
@@ -210,7 +210,7 @@ def add_tags(data_catalog: AwsAthenaDataCatalog) -> None:
                 data_catalog.tags = bend(S("Tags", default=[]) >> ToDict(), tags[0])
 
         for js in json:
-            if (name := js.get("CatalogName")) is not None and isinstance(name, str):
+            if (name := js.get("CatalogName")) is not None and isinstance(name, str) and name != "AwsDataCatalog":
```

Comment:
Can you add a comment that explains why this catalog is sorted out?
Should we sort out all catalogs of type `GLUE`?

Reply:
> Should we sort out all catalogs of type GLUE?

I don't think so.